docs/static/v1/ce-paste-theme.css

```diff
@@ -135,10 +135,28 @@ input[readonly="true"] {
   background-color: rgb(244, 244, 246);
 }
 
+input[type='submit'] {
+  cursor: pointer;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+  padding-left: 0.75rem;
+  padding-right: 0.75rem;
+  border: 1px solid;
+  border-radius: 4px;
+  margin-top: 1em;
+  font-size: 0.875rem;
+  line-height: 1.25rem;
+}
+
 input.phone-number {
   width: 15rem;
 }
 
+input.intl-tel-input {
```

Comment:
should this be instead input[type="tel"] rather than relying on specific classes?

Reply:
Good idea -- I checked and our only other apps that use `type="tel"` don't use `ce-paste-theme` yet, so this is a safe change.